mozci/configuration.py

```diff
@@ -109,6 +109,10 @@ class Configuration(Mapping):
     ) or os.environ.get("TASKCLUSTER_SECRET")
     DEFAULTS = {
         "merge": {
+            "autoclassification": {
+                "enabled": False,
+                "test-suite-names": [],
+            },
```

Comment:
Could you also update the docs please?
https://github.com/mozilla/mozci/blob/master/docs/configuration.rst

Reply:
Done !